src/test/suite/fixtures/recorded/compoundTargets/clearPastStartOfToken.yml

```diff
@@ -1,11 +1,11 @@
-languageId: typescript
+languageId: plaintext
 command:
   version: 1
-  spokenForm: take past start of token
-  action: setSelection
+  spokenForm: clear past start of token
+  action: clearAndSetSelection
   targets:
     - type: range
-      start: {type: primitive}
+      start: {type: primitive, isImplicit: true}
```

Comment:
do we need to do a migration?  what happens when they issue the command with new extension / old Talon?  Does this no longer work as it did before?

Reply:
Should work as before

Reply:
Needs a migration 😕.  I can knock it out